README.md

```diff
@@ -11,6 +11,7 @@ NEAR Wallet Selector makes it easy for users to interact with your dApp by provi
 - [Ledger](https://www.npmjs.com/package/@near-wallet-selector/ledger) - Hardware wallet.
 - [WalletConnect](https://www.npmjs.com/package/@near-wallet-selector/wallet-connect) - Bridge wallet.
 - [Nightly Connect](https://www.npmjs.com/package/@near-wallet-selector/nightly-connect) - Bridge wallet.
+- [NearFi Wallet](https://www.npmjs.com/package/@near-wallet-selector/nearfi) - Injected wallet.
```

Comment:
I'm not sure if this is an injected wallet. This is more of a mobile wallet.

Reply:
Just not sure how the injected is defined, but yeah I'm agree NearFi is a mobile wallet